op-node/rollup/types.go

```diff
@@ -50,6 +50,17 @@ type Genesis struct {
 	SystemConfig eth.SystemConfig `json:"system_config"`
 }
 
+type PlasmaConfig struct {
+	// L1 DataAvailabilityChallenge contract proxy address
+	DAChallengeAddress common.Address `json:"da_challenge_contract_address,omitempty"`
```

Comment:
Did you try changing the type to a pointer to get the `omitempty` to work correctly ?
```
DAChallengeAddress *common.Address `json:"da_challenge_contract_address,omitempty"`
```

Reply:
No, but I'm not super worried about `omitempty` here